scripts/test-index.sh

```
@@ -0,0 +1,83 @@
+  ! -path "*/.git/*" ! -path "*/node_modules/*" ! -name "INDEX.md" \
+  | sort | while read -r file; do
+
+  # Skip files that already have a summary
+  if head -1 "$file" | grep -q '^---$' && sed -n '2,/^---$/p' "$file" | grep -q '^summary:'; then
+    continue
+  fi
+
+  count=$((count + 1))
+  [ "$count" -gt "$LIMIT" ] && break
+
+  relpath="${file#$REPO/}"
```

Shellcheck (0.11.0)

[info] Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

(SC2295)